cluster-autoscaler/cloudprovider/azure/azure_scale_set_test.go

```diff
@@ -23,23 +23,40 @@ import (
 
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2022-08-01/compute"
 	"github.com/Azure/go-autorest/autorest/to"
-	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
 	"sigs.k8s.io/cloud-provider-azure/pkg/azureclients/vmclient/mockvmclient"
 	"sigs.k8s.io/cloud-provider-azure/pkg/azureclients/vmssclient/mockvmssclient"
 	"sigs.k8s.io/cloud-provider-azure/pkg/azureclients/vmssvmclient/mockvmssvmclient"
 )
 
+const (
+	testLocation = "eastus"
```

Comment:
where is the testASG const defined?
`testASG      = "test-asg"`

Reply:
I think it is from azure_manager_test.go

Reply:
I agree and am fine with not changing it rn, but I just hate that that's how it's been 😂 (hence the thumbs down)